### PR TITLE
fix #12188 integration-cli: use "top" instead of "sleep" to avoid timeout

### LINKED ISSUE #12188: integration-cli: Tests using sleep can timeout too early - change to "top" instead
There is a lot of testing code that is using "docker run busybox sleep N", where N is usually one of (2, 10, 20), with a subsequent docker call to test something on this container (like docker ps or docker exec .. ping).

The problem with this is that in certain environments, the docker sleep process may have ended before the test can finish, resulting in a test failure (unexpected output).

I'd like to increase the sleep time (perhaps to 1000 - and perhaps storing it as a constant somewhere), so that it avoids the slow environment situation and runs the tests correctly.

Does anyone have an issue with such a change?

### PR DESCRIPTION
**Pull request by twhiteman**
Updated most of the sleep tests (there's still a few left where sleep is applicable).

Fix #12188 

Signed-off-by: Todd Whiteman todd.whiteman@joyent.com

### CODE REVIEW

File: integration-cli/docker_cli_kill_test.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func TestKillContainer(t *testing.T) {
-	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "sh", "-c", "sleep 10")
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "sh", "-c", "top")
```
Comment (LK4D4) [inline integration-cli/docker_cli_kill_test.go]: You can use just `top` in such cases. Without `"sh", "-c"`

**Comment by LK4D4** [comment]
@twhiteman It is better to squash your commits. You can do it with `git rebase -i`

**Comment by LK4D4** [comment]
Overall changes are very good.

**Comment by icecrime** [comment]
+1 for squashing, otherwise LGTM

**Pull request closed without merging**